patient-appointment-management/functions/auth.private.js

```diff
@@ -1,19 +1,77 @@
-/* eslint-disable no-negated-condition, no-else-return */
+/* eslint-disable no-negated-condition, no-else-return, import/newline-after-import, no-use-before-define, vars-on-top, no-var, object-shorthand, sonarjs/prefer-immediate-return */
```

Comment:
Please remove this line and adjust the code instead to be in accordance with our coding standards.

Reply:
will ask the developer to look at this one.

Reply:
Dominik, the changes are being made for this right now so we can meet the HIMSS deadline.

Reply:
changes have been made to remove any  eslint-disable parameters introduced in this PR. There are some parameters from before that we did not change at this time